spidermon/contrib/actions/email/__init__.py

```diff
@@ -59,7 +59,7 @@ def __init__(
             raise NotConfigured(
                 "You must provide at least one recipient for the message."
             )
-        if not self.subject:
+        if not self.subject and not self.subject_template:
             raise NotConfigured("You must provide a subject for the message.")
```

Comment:
What about updating the message in `NotConfigured` to say `a subject or a subject template`?

Reply:
```suggestion
            raise NotConfigured("You must provide a subject or a subject-template for the message.")
```
Reason I did not include that was a similar message for the setting the body: "you must provide a body for...". But your suggestion makes sense of course.
Wouldn't it make sense to use the names of the settings here? Else it's necessary to guess/look-up the name of the setting. But I'm fine with either one tbh.

It's my first time using a suggestion, is this enough to pick up the change and resolve this?

Reply:
I also like the idea of including the setting names, for example between parentheses after each concept (subject, subject template).

By the way, I don’t think ‘subject-template’ is necessary or correct English. [Hyphens can be used for joining words](https://en.wikipedia.org/wiki/Hyphen#Joining), but I don’t think this is the right context for that. It would be correct however for ‘subject-defining template’ or ‘subject-*templater*’.

As for suggestions, you can pick the change to quickly apply it to your branch, but you can also simply perform the change yourself on a new commit.

Reply:
I liked the idea of including the related spidermon setting inside parenthesis in this message, so the error message will be much more helpful. I believe that `subject template` (without hyphen) is the right way to mention this setting.